src/utils/themeStyles/index.ts

```diff
@@ -42,3 +42,6 @@ export const setThemeFontWeight = (theme: ThemeType) =>
   theme.__type === 'secondary'
     ? theme.TYPOGRAPHY.fontWeight.normal
     : theme.TYPOGRAPHY.fontWeight.bold;
+
+export const applyVerticalAlignmentOffset = (theme: ThemeType, offset = '1') =>
```

Comment:
I would consider re-naming this so the intent is more obvious to downstream users who aren't familiar with [all the details of the root cause](https://github.com/curology/radiance-ui/pull/720#discussion_r568047415). 

Perhaps `applyPrimaryThemeVerticalOffset`. I think it's good for the name to refer to the theme explicitly (so more like `setSecondaryHeadingFont` and less like `setButtonStyleFontWeight`). I mentioned `PrimaryTheme` explicitly in it since `setSecondaryHeadingFont` is only ever used within `radiance-ui` so the intent is more obvious, but since this will be used in consumer apps I think the more verbose the better. I removed `Alignment` in case `applyPrimaryThemeVerticalAlignmentOffset` was *too* verbose, though.

Reply:
yep no problem, renamed and publishing